cli/pipeline/template.go

```diff
@@ -191,7 +192,8 @@ func NewPodDefinition(pipelineDefinition *PipelineDefinition, pipelineDefinition
 	stepName := sanitizeName(pipelineDefinitionStep.Step)
 	branchName := sanitizeName(pipelineDefinitionStep.Branch)
 	stepVersion := sanitizeName(pipelineDefinitionStep.Version)
-	podName := fmt.Sprintf("%s-%s-%s-%s", sanitizeName(pipelineDefinition.Pipeline), sanitizeName(pipelineDefinitionStep.Version), stepName, branchName)
+	timestamp := time.Now().UTC()
+	podName := fmt.Sprintf("%s-%s-%s-%s-%s", sanitizeName(pipelineDefinition.Pipeline), sanitizeName(pipelineDefinitionStep.Version), stepName, branchName, timestamp.String())
```

Comment:
lets check the max length of pod names - to make sure we avoid running into those limits. 

I think if you just add the last 4 digits of the integer time - you'd very rarely run into collisions.

Reply:
The limit is 253 characters. Will modify by having the last 4 digits then